samples/openapi3/client/petstore/python-experimental/tests/test_api_validation.py

```diff
@@ -44,6 +46,27 @@ def test_configuration(self):
             config.disabled_client_side_validations = 'foo'
         config.disabled_client_side_validations = ""
 
+    def test_servers(self):
+        config = petstore_api.Configuration(server_index=1, server_variables={'version': 'v1'})
+        api_client = petstore_api.ApiClient(config)
+        api = pet_api.PetApi(api_client)
+
+        def request(expected_url, method, url, **kwargs):
+            assert expected_url == url
+            raise RuntimeError('pass')
+
+        api_client.request = functools.partial(request, 'http://path-server-test.petstore.local/v2/pet')
+        try:
+            api.add_pet({'name': 'pet', 'photo_urls': []})
+        except RuntimeError as e:
+            assert "pass" == str(e)
+
+        api_client.request = functools.partial(request, 'https://localhost:8080/v1/pet/123456789')
+        try:
+            api.delete_pet(123456789)
+        except RuntimeError as e:
+            assert "pass" == str(e)
```

Comment:
Please also add tests of server_operation_index and server_variables.
That code is added in this PR and does not yet have any test coverage.

Reply:
I have created a separate package with a detailed testsuite.